matter_server/client/client.py

```diff
@@ -182,7 +182,7 @@ async def set_thread_operational_dataset(self, dataset: str) -> None:
     async def open_commissioning_window(
         self,
         node_id: int,
-        timeout: int = 300,
+        timeout: int = 300,  # noqa: ASYNC109 timeout parameter required for native timeout
```

Comment:
let's just ignore this rule in the pyproject as (imo) its a weird rule

Reply:
For pure Python function using `asyncio.timeout()` and handling `asyncio.CancelledError` is better over a parameter, so for these cases the rule does make sense IMHO. 

 All our use cases do have a good reason to have the timeout parameter, but it could be that someone introduces another case. Disabling globally would not catch that. So I'd prefer going with explicit ignores.